apeer_dev_kit/_core.py

```diff
@@ -1,59 +1,71 @@
-﻿import os
+import os
 import json
+import logging as log
 
 from ._utility import copyfile
 
 class _core:
     def __init__(self):
+        log.basicConfig(format='%(asctime)s [ADK:%(levelname)s] %(message)s', level=log.INFO, datefmt='%Y-%m-%d %H:%M:%S')
         try:
+            log.info('Initializing')
             self._outputs = {}
             self._wfe_output_params_file = ''
             self._input_json = json.loads(os.environ['WFE_INPUT_JSON'])
+            log.info('Found environment variable WFE_INPUT_JSON to be {}'.format(self._input_json))
```

Comment:
I think it is better not to write "WFE_INPUT_JSON" in log. We should abstract this information from normal user. I think better message would be something like `'Found inputs to be {}'` or something else like this